From dd5beeace98f9b58049f8c9951ae4a33b87ded02 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Fri, 20 Sep 2019 01:12:11 +0300
Subject: [PATCH] tinycompress: Conditionally enable extended compress format

This is squashed with:

Author: LuK1337 <priv.luk@gmail.com>
Date:   Thu Oct 15 17:12:12 2020 +0200

    tinycompress: Convert lineage product variables to soong config variables

    Change-Id: I5c0d32b9def67979c74a49d9dc03415a5cc4d2d1

Change-Id: Iac589fcc06bd7cea544475f7d82eaf4af8af2348
---
 Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Android.bp b/Android.bp
index a4f8d6d..98fbd16 100644
--- a/Android.bp
+++ b/Android.bp
@@ -21,6 +21,7 @@ license {
 
 cc_library_shared {
     name: "libtinycompress",
+    defaults: ["extended_compress_format_defaults"],
     vendor: true,
 
     cflags: [
-- 
2.30.2

